docker-init-script/out/test/resources/eu/xenit/docker/alfresco/defaults/4.2.properties

```diff
@@ -0,0 +1,9 @@
+index.subsystem.name=solr
```

Comment:
You seem to have committed a wrong directory here containing classes and resources.

Is your intellij wrongly configured not to delegate to Gradle?

Reply:
Correct.